apps/site/components/LandingPage/TrackItem.tsx

```diff
@@ -0,0 +1,13 @@
+import TrackPlayer from "../TrackPlayer"
+import AddTrackButton from "./AddTrackButton"
+
+const TrackItem = ({ track }: any) => {
+  return (
+    <div className="space-y-2 relative">
+      <TrackPlayer track={track} />
+      <AddTrackButton />
+    </div>
+  )
+}
```

Comment:
return is redundant right?
```suggestion
const TrackItem = ({ track }: any) =>  (
    <div className="space-y-2 relative">
      <TrackPlayer track={track} />
      <AddTrackButton />
    </div>
  )
```